Tests/RxFeedbackTests/FeedbackLoopsTests.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  FeedbackLoopsTests.swift
+//  RxFeedback
+//
+//  Created by Alexander Sokol on 23/09/2017.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+import XCTest
+import RxFeedback
+import RxSwift
+import RxTest
+
+class FeedbackLoopsTests: XCTestCase {
+    let bag = DisposeBag()
+}
+
+// Tests on the react function with not an equatable or hashable Control.
+extension FeedbackLoopsTests {
+
+    func testIntialNil() {
```

Comment:
I think test names need to be a bit more descriptive maybe. I'm not sure what is being tested.